Package.swift

```diff
@@ -1,6 +1,23 @@
 // swift-tools-version:4.2
 import PackageDescription
 
+let compilerSpecificDependencies: [Package.Dependency]
+#if compiler(>=5)
+compilerSpecificDependencies = [
+    .package(url: "https://github.com/antitypical/Result.git", from: "5.0.0"),
+    .package(url: "https://github.com/Carthage/ReactiveTask.git", .branch("release-0.17.0")),
```

Comment:
`release-0.17.0` is the branch of https://github.com/Carthage/ReactiveTask/pull/120, which has been open for a year :)

AFAICT, nothing is preventing that PR from being merged, there just hasn't been a reason to yet. It needs to get merged before this PR, since [versioned packages can't have branch dependencies](https://github.com/apple/swift-package-manager/blob/master/Documentation/PackageDescription.md#package-dependency-requirement).